src/utilities/buildASTSchema.js

```diff
@@ -342,7 +344,7 @@ export class ASTDefinitionBuilder {
       case Kind.INPUT_OBJECT_TYPE_DEFINITION:
         return this._makeInputObjectDef(def);
       default:
-        throw new Error(`Type kind "${def.kind}" not supported.`);
+        invariant(false, `Type kind "${def.kind}" not supported.`);
```

Comment:
While the above case should probably be `invariant`, this one should almost definitely me `throw new GraphQLError`: you've got a type definition that somehow parsed but at validate time is invalid. In this instance it's very helpful to see where in your document that error actually is.

Reply:
@mjmahone You right 👍 especially tanking into account possible Experimental features.

Reply:
@mjmahone Fixed.